### DISCUSSION TITLE
No Unicode Mapping Warning

### DISCUSSION THREAD

**Discussion by smyaseen** [red6/pdfcompare#107] [Q&A] [answered]
Hi, 

While reading a PDF, I'm getting the "No Unicode mapping..." warning. I want to ask if it's ok to ignore this, or there is a problem? What can I do to solve this? I can't share the PDF due to confidentiality. 

![image](https://user-images.githubusercontent.com/37702376/123630607-2a51d280-d82f-11eb-8e89-32a6a547368c.png)

**Comment by finsterwalder** [comment, marked as answer]
I can't really help here. This is a message from the underlying PdfBox library, that I'm using to render the PDFs.
It basically tells you, that the used font does not have a proper mapping for some characters. Most likely a fallback font will be used, but please check with PdfBox: https://pdfbox.apache.org/

**Reply by smyaseen** [reply]
I understand. Thank you.

**Reply by smyaseen** [reply]
One thing, is it ok to ignore this warning?

**Reply by finsterwalder** [reply]
Like I said, I'm not an expert. And i don't know the impact.
Best case: Nothing to worry about.
Worst case: chars are not rendered and you don't see differences in those chars.

**Reply by smyaseen** [reply]
Alright thank you.
